File: src/main/java/com/haozileung/web/service/permission/IUserService.java
package com.haozileung.web.service.permission;

import com.haozileung.infra.pager.PageResult;
import com.haozileung.web.domain.permission.User;

public interface IUserService {

    void save(User dictionary);

    void update(User dictionary);

    void delete(User dictionary);

    PageResult<User> query(User dictionary, int pageNo, int pageSize);
}

File: src/main/kotlin/com/haozileung/web/service/permission/IUserService.kt
package com.haozileung.web.service.permission

import com.haozileung.infra.pager.PageResult
import com.haozileung.web.domain.permission.User

interface IUserService {

    fun save(dictionary: User)

    fun update(dictionary: User)

    fun delete(dictionary: User)

    fun query(dictionary: User, pageNo: Int, pageSize: Int): PageResult<User>
}
